src/wstool/multiproject_cli.py

```diff
@@ -61,13 +62,14 @@
     "remove":   "remove an entry from your workspace config, without deleting files",
     "snapshot": "write a file specifying repositories to have the version they currently have",
     "diff":     "print a diff over some SCM controlled entries",
+    "foreach":  "run shell command in each entries",
```

Comment:
In #29 You said
~~~
wstool foreach [--bzr] [--git] [--hg] [--svn] [repository]+ [--shell] [--us] [--hide-stdout] [--timeout=double] [..no-filter]
~~~
Do you want arguments which give entries the command should be run in?
so, first argument -> command
rest argument -> entries?